cognite_toolkit/examples/cdf_oid_example_data/default.config.yaml

```diff
@@ -1,3 +1,10 @@
 # Only valid for this module, loads template variables from environment
-raw_db: Valhall_System_23
-data_set: Valhall_System_23
+# RAW DB name structure: <data type>_<location>_<source>  ( see README for name standard)
+# 
+# In the example below we are setting up a project based on the Open Industry Data (OID), 
+# that originates from the Valhall oil rig. Hence the example location below is oid
+#
+raw_db_asset: asset_oid_workmate
```

Comment:
So to illustrate my comment further below on how you could structure this more enforced:
In default.config.yaml:
```
location: oid
source_asset: workmate
```

And then in the config yaml file for each raw table:
```
dbName: asset_{{location}}_{{source_asset}}
tableName: asset2children
```

Reply:
sure - then this is fine for assets